3rdparty/protobuf/bazelbuild_remote-apis/README.md

```diff
@@ -1,12 +1,3 @@
 This is a dump of the .proto files from https://github.com/bazelbuild/remote-apis directory build.
 
 This dump was taken at git sha cbf6ada7f5b2a0ce14646bf983d03b49118f0ec8.
-
-The following script was run to enable Bytes fields for Rust:
-```
-sed -i '' '/^package /a\
-\
-import "rustproto.proto";\
-option (rustproto.carllerche_bytes_for_bytes_all) = true;\
-' build/bazel/remote/execution/v2/remote_execution.proto
-```
```

Comment:
If this is just the contents of a remote repo with some specific programmable edits, it sounds like cloning the repo itself in a build script (possibly using the tiniest of subcrates) might be reasonable? I understand being wary of trying to abstract too early and I don't have an issue with this as-is.

Reply:
Sounds like I need to work on the rust/cargo ruleset :)